state/API.F90

```diff
@@ -1,6 +1,6 @@
 module mapl3g_State_API
 
-   use mapl3g_StateGetBundle, only: MAPL_StateGet => StateGetBundle
+   use mapl3g_StateGet, only: MAPL_StateGet => StateGet
```

Comment:
Not going to push it, but ... in theory regular procedures should be snake case in MAPL.   But is clearly a bit of a mismash with legacy and ESMF influences.